src/autoIncrement.ts

```diff
@@ -92,20 +93,31 @@ export function AutoIncrementID(schema: mongoose.Schema<any>, options: AutoIncre
       return;
     }
 
+    const modelName = (this.constructor as any).modelName;
+    logger.info('modelName "%s"', modelName);
```

Comment:
maybe remove this debug log again, or make it have more sense

Reply:
will remove